.github/workflows/ci.yaml

```diff
@@ -19,63 +19,77 @@ jobs:
           - '3.8'
           - '3.9'
     runs-on: ${{ matrix.os }}
-    defaults:
-      run:
-        shell: bash -l -eo pipefail {0}
     steps:
-      - uses: actions/checkout@v2
-      - uses: conda-incubator/setup-miniconda@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python }}
-          mamba-version: "*"
-          channels: conda-forge,bioconda
-          channel-priority: true
-          activate-environment: test
-      - run: cat ~/.profile || true
-      - run: cat ~/.bashrc || true
-
-      - name: Set up test environment
-        run: |
-          mamba install augur auspice snakemake
-          # The small build we'll use as an integration test case.
-          git clone https://github.com/nextstrain/zika-tutorial
 
       - name: Install Nextstrain CLI
         run: python3 -m pip install --upgrade '.[dev]'
 
-      - run: conda info
-      - run: conda list
+      - name: Run pytest
+        run: pytest -v
 
-      - name: Check python version
-        run: |
-          # Assert that we're on the expected Python version, in case the GH
-          # Actions environment is messed up.
-          python3 --version | grep -F 'Python ${{ matrix.python }}.'
+  build-dist:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
```

Comment:
Could this be kept consistent with `*.matrix.python` and the version used in the `doc` job? More concretely, 2 suggestions:

1. Add `3.10` to `test-source.matrix.python`, `test-dist.matrix.python`
2. Update `doc` job to use `3.10`

Reply:
I'll add 3.10 to the two version matrices. ISTR some issue with setup-miniconda and 3.10, but it's a very vague recollection so we'll find out.

The doc job uses 3.6 because that's what `.readthedocs.yml` specifies. I suppose I can bump that up too and then bump the `doc` CI job.

Reply:
Ok, a bit of turbulence to get thru.

- Conda indeed has problems with 3.10, reporting: `package augur-10.0.0-py_0 requires bcbio-gff >=0.6.0, but none of the providers can be installed`. I'm going to drop the `test-dist` jobs on 3.10 because of this, but keep the `test-source` jobs.
- [RTD complains](https://readthedocs.org/projects/nextstrain-cli/builds/17121136/) about 3.10 saying that it's not a supported version. Highest is 3.8.
- However, the [config spec now suggests a different field](https://docs.readthedocs.io/en/stable/config-file/v2.html#python-version) to use for setting up Python, so I'll look into that.

Reply:
https://github.com/nextstrain/cli/pull/180/commits/02af3a1839fd10c09e3c79594caeb9b3f0550fc8

https://github.com/nextstrain/cli/pull/180/commits/d12f5b5bfe65e527bdf2cf050c06dc8fcc6295ac